migrations/capcons/storagecapmigration.go

```diff
@@ -129,18 +131,14 @@ func IssueAccountCapabilities(
 			borrowType = capabilityBorrowType.(*interpreter.ReferenceStaticType)
 
 		} else {
-			if _, _, ok := untypedCapabilityMapping.Get(addressPath); ok {
-				continue
-			}
-
```

Comment:
Removed this, because me might want to get a count on how many such capabilities are there, even if they point to the same storage location.

Reply:
I don't think we can remove this, as otherwise one controller per capability is issued.
Maybe add separate reporting before the bail-out

Reply:
oh I see!

Reply:
Updated it to always log (https://github.com/onflow/cadence/pull/3524/commits/0f12301ee4a7ee4cb8db24aca684d66a6e17440b). Then any cap that did not get inferred can be found by diffing the two logs entries.